packages/terra-application-links/examples/test-examples/config/testLinkConfig.js

```diff
@@ -1,7 +1,7 @@
 const testLinkConfig = [
   { path: '/test1', text: 'test 1' },
   { path: '/test2', text: 'test 2' },
-  { path: '/test3', text: 'test 3' },
+  { path: '/test3', text: 'test 3', hasSubMenu: true },
```

Comment:
I know you are adding tests, but can we get a wdio test that captures the link with a submenu?

Reply:
Bump.